src/ert/scheduler/driver.py

```diff
@@ -48,3 +45,11 @@ async def kill(self, iens: int) -> None:
     @abstractmethod
     async def poll(self) -> None:
         """Poll for new job events"""
+
+    @abstractmethod
+    async def wait(self, iens: int) -> None:
+        """Blocks the execution of a job associated with a realization.
```

Comment:
I think it is too vague what is to be waited for by in implementations of this function. And should we use "block", as we never block anything, do we?

Reply:
We do when we call block, but you are correct it's not the job execution but the event loop only  I guess